language/en/phpbb_ideas_acp.php

```diff
@@ -22,6 +22,8 @@
 	// ACP module
 	'ACP_IDEAS_FORUM_ID'			=> 'Ideas forum',
 	'ACP_IDEAS_FORUM_ID_EXPLAIN'	=> 'Select the forum that Ideas topics will be posted to.',
+	'ACP_IDEAS_FORUM_SETUP'			=> 'Apply ideas forum settings',
```

Comment:
What do you think? Like on the main page. The button says `Run now`. The Line here is `Set up the Ideas forum`.